privacy-security-explainer.md

```diff
@@ -0,0 +1,68 @@
+# Privacy and security
+
+The WebXR Device API enables developers to build content for AR and VR hardware that uses one or more sensors to infer information about the real world, and may then present information about the real world either to developers or directly to the end user. In such systems there are a wide range of input sensor types used (cameras, accelerometers, etc), and a range of real-world data generated. This data is what allows web developers to author WebXR-based experiences. It also enables developers to infer information about users such as profiling them, fingerprinting their device, and input sniffing. Due to the nature of the Web, WebXR has a higher responsibility to protect users from malicious data usage than XR experiences delivered through closed ecosystem app stores.
```

Comment:
Totally agree that we're in this weird middle ground right now since we haven't officially split AR stuff into a separate module.  Given that `immersive-ar` is still in the spec, I'm hesitant to remove references to it (as you'll see in a subsequent PR).  But *yes* I absolutely agree that as it stands, we don't yet have a cohesive story for AR-related threat vectors.
For now, are you ok with my leaving the AR references under the knowledge that there will be a full "culling" of all references when we make the official module split?

Reply:
I'm OK with that, it's not a blocking concern for now and can be addressed later.